src/anomalib/models/reverse_distillation/config.yaml

```diff
@@ -35,21 +35,20 @@ model:
     - layer1
     - layer2
     - layer3
-  early_stopping:
-    patience: 3
-    metric: pixel_AUROC
-    mode: max
+  # early_stopping: # optional
+  #   patience: 3
+  #   metric: pixel_AUROC
+  #   mode: max
```

Comment:
We could either increase the patience to act like there is no early stopping. Or we could remove this all together. Commented out code/config might be confusing to some

Reply:
I think it's better to remove it from the config, it looks like 200 epochs give good results for the variety of tasks. If I remember it correctly, training RD for 200 epochs achieved 96 AUROC for images at VisA.